DoLah/Commands/Command.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include <vector>
+#include <string>
+#include "..\Models\Task.h"
+
+namespace DoLah {
+
+	class Calendar;
+
+	class AbstractCommand {
```

Comment:
C++ doesn't have interfaces, so in place of that they have abstract classes. Correct me if I'm wrong.

I'm changing ITaskCommand class name to AbstractCommand in line with ITask->AbstractTask.